index.html

```diff
@@ -238,8 +238,11 @@ <h3 class="contact-header">Contact me</h3>
       or a project that needs coding. I'd love to help with it </p>
    
     <form action="https://formspree.io/f/mzbwzdgd" class="form-container" method="post" onsubmit="return formSubmit();">
```

Comment:
- [ ] Kindly make sure you stick to the design 
  Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193887569-56ffe5cb-eb54-423a-981a-db3899ef4c51.png)

Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193887741-eae21d44-c84f-441e-aa56-f54a4728396d.png)